### PR TITLE
Bratseth/inputs constants

### PR DESCRIPTION
**Pull request by bratseth**
@kkraune or @jobergum please review

### CODE REVIEW

**Comment by jobergum** [comment]
> Hence the expression: as greedy as a pig

https://www.youtube.com/watch?v=2xUynRdzzsM

We need to wait until an open release with the fixes for inputs scalars,  I see test failures in the doc testing for it.

File: en/proton.html [hunk 1]
```diff
@@ -552,7 +552,7 @@ <h2 id="proton-maintenance-jobs">Proton maintenance jobs</h2>
   <tr>
     <th>CPU</th>
     <td>CPU similar to feeding.
-        Consumes capacity from the index write thread - hence has feeding impact</td>
+        Consumes capacity from the index write threadm, so has feeding impact</td>
```
Comment (kkraune) [inline en/proton.html:555]:
```suggestion
        Consumes capacity from the index write thread, so has feeding impact</td>
```

**Comment by bratseth** [comment]
Yes, that'll be in Monday's release.

Out hence incidence is now down to just 10x normal ...

**Review by kkraune — APPROVED** [review]
Great!

**Pull request merged**